test/test_torch.py

```diff
@@ -7045,6 +7045,56 @@ def test_cat_out_channels_last(self, device):
         res2 = torch.cat((x, y), out=z)
         self.assertEqual(res1, res2)
 
+    def test_cat_in_channels_last(self, device):
```

Comment:
Please move test to other class where `device` is unnecessary or add cpu only decorator or use `device` appropriatly.